app/services/ordnance/address_lookup_results.rb

```diff
@@ -5,14 +5,10 @@ class AddressLookupResults
     DEFAULT_COUNTRY = 'UNITED KINGDOM'.freeze
     OTHER_TERRITORIES = ['ISLE OF MAN', 'JERSEY', 'GUERNSEY'].freeze
 
-    Address = Struct.new(:address_line_one, :address_line_two, :city, :country, :postcode) do
+    Address = Struct.new(:address_line_one, :address_line_two, :city, :country, :postcode, :lookup_id) do
```

Comment:
Hmm, yep I considered that, but then to save 4 lines in this class, you would end up with a separate class much bigger as you would need attribute readers for each of the fields, plus the initialiser to take them, etc.

I think if we ever need to add more methods or some complex logic it would make sense, but for now is pretty much a data structure to hold raw data so a Struct fits well. Also nothing more PORO than a good old Struct 😄